src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -1708,6 +1719,11 @@ public Void caseFunctionType(FunctionType type) {
     }
   }
 
+  private boolean isAFunctionPrototypeProp(String propName) {
+    return propName.equals("apply") || propName.equals("call") ||
```

Comment:
This is the complete list, that I was seeing emitted. My guess would be those should be filtered, because we asked for get*Own*PropertyNames, but somehow they still appear.